nussl/core/augmentation.py

```diff
@@ -0,0 +1,377 @@
+        "decays": make_arglist_ffmpeg(decays),
+        "depths": make_arglist_ffmpeg(depths)
+    }
+
+    return apply_ffmpeg_filter(audio_signal, "chorus", **filter_kwargs)
+
+def phaser(audio_signal, in_gain=.4, out_gain=.74, delay=3, 
+        decay=.4, speed=.5, _type="triangular"):
+    """
+    https://ffmpeg.org/ffmpeg-all.html#aphaser
+
+    Applies Phaser Filter
+    Args:
+        audio_signal: An AudioSignal object
+        TODO: Write this documentation.
+    Returns:
+        augmented_signal: A copy of the original audio signal, with augmentations applied
+    """
+    # TODO: Arg checks
+    filter_kwargs = {
+        "in_gain": in_gain,
+        "out_gain": out_gain,
+        "delay": delay,
+        "speed": speed, 
+        "delay": delay,
+        "type": _type
+    }
+
+    return apply_ffmpeg_filter(audio_signal, "aphaser", **filter_kwargs)
+
+def flanger(audio_signal, delay=0, depth=2, regen=0, width=71, 
+    speed=.5, phase=25, shape="sinusoidal", interp="linear"):
+    """
+    https://ffmpeg.org/ffmpeg-all.html#flanger
+
+    Args:
+        audio_signal: An AudioSignal object
+        TODO: Write this documentation.
+    Returns:
+        augmented_signal: A copy of the original audio signal, with augmentations applied
+    """
+    # TODO: Arg checks
+    filter_kwargs = {
+        "delay": delay,
+        "depth": depth,
+        "regen": regen,
+        "width": width,
+        "speed": speed, 
+        "phase": phase, 
+        "shape": shape, 
+        "interp": interp
+    }
+
+    return apply_ffmpeg_filter(audio_signal, "flanger", **filter_kwargs)
+
+def emphasis(audio_signal, level_in, level_out, _type, mode='production'):
+    """
+    https://ffmpeg.org/ffmpeg-all.html#aemphasis
+
+    Applies the emphasis filter, which either creates or restores 
+    signals from various physical mediums. 
+    Args:
+        audio_signal: An AudioSignal object
+        level_in: Input gain
+        level_out: Output gain
+        _type: medium type to convert/deconvert from 
+        mode: reproduction to convert from physical medium, 
+            production to convert to physical medium.
+
+    Returns:
+        augmented_signal: A copy of the original audio signal, with augmentations applied 
+    """
+    if level_in < LEVEL_MIN or level_in > LEVEL_MAX \
+        or level_out < LEVEL_MIN or level_out > LEVEL_MAX:
+        raise ValueError(f"level_in and level_out must both be between {LEVEL_MIN} AND {LEVEL_MAX}")
+
+    filter_kwargs = {
+        'level_in': level_in,
+        'level_out': level_out,
+        'mode': mode,
+        'type': _type
+    }
+
+    return apply_ffmpeg_filter(audio_signal, "aemphasis", **filter_kwargs)
+
+def _compressor_argcheck(level_in, mode, reduction_ratio,
+    attack, release, makeup, knee, link,
+    detection, mix, threshold):
+
+    # The following values are taken from ffmpeg documentation
+    if level_in < LEVEL_MIN or level_in > LEVEL_MAX or \
```

Comment:
Use something more like

```
if (
   level_in < LEVEL_MIN or level_in > LEVEL_MAX or
   mode not in {"upward", "downward"} or 
)
```

Parens instead of \ for line breaks.